p/scripts/main.js

```diff
@@ -252,7 +252,7 @@ function toggleContent(new_active, old_active, skipping = false) {
 	}
 
 	if (old_active[0] !== new_active[0]) {
-		if (isCollapsed && !skipping) { // BUG?: isCollapsed can only ever be true 
+		if (isCollapsed && !skipping) { // BUG?: isCollapsed can only ever be true
```

Comment:
The function declaration a bit higher up contains a default parameter, which is ES6.
```javascript
function toggleContent(new_active, old_active, skipping = false) { }
```
This might not be compatible with all the browsers we support so far (e.g. IE11?)
I suggest removing the default parameter, and being explicit in all the calls